MoreLinq/StartsWith.cs

```diff
@@ -0,0 +1,49 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2012 Atif Aziz. All rights reserved.
```

Comment:
Feel free to Copyright this file (including related files) and (presumably original) work to you, including setting the first year of publication to presumably (2016). If you've taken inspiration from a publicly available implementation, then a common courtesy is to leave a comment crediting the source (assuming license compatibility).